webserver/src/app/api/v0/wordpress/login/login.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
`component.spec.ts` should be removed...

Reply:
Removed in #35